robotools/evotools/exceptions.py

```diff
@@ -0,0 +1,7 @@
+"""Exceptions related to liquid handling with worklists."""
+
+__all__ = ("InvalidOperationError",)
+
+
+class InvalidOperationError(Exception):
```

Comment:
This is a very small module. Maybe you could have all exceptions on a higher level with evotools / liquidhandling, but it's also ok like this.

Reply:
The idea behind this design decision (from years ago) was to add support for other, non-EVOware robots, which is still something I'd like to explore further at some point

Thanks for reviewing!